CMakeLists.txt

```diff
@@ -235,8 +236,7 @@ if (FMT_INSTALL)
   include(GNUInstallDirs)
   include(CMakePackageConfigHelpers)
   set_verbose(FMT_CMAKE_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/fmt CACHE STRING
-              "Installation directory for cmake files, relative to "
```

Comment:
I'd change "relative to ${CMAKE_INSTALL_PREFIX}" to "absolute or relative to ${CMAKE_INSTALL_PREFIX}" instead of removing this part completely. Same in other places.

Reply:
Would wording like “absolute path, or relative path that will be joined to ${CMAKE_INSTALL_PREFIX}”? I want to make it clear that the absolute paths can be arbitrary, not only based on `${CMAKE_INSTALL_PREFIX}`.